client_dashboard/common/src/commonMain/kotlin/org/thechance/common/App.kt

```diff
@@ -1,9 +1,21 @@
 package org.thechance.common
 
-import com.beepbeep.designSystem.DesignApp
+import androidx.compose.animation.ExperimentalAnimationApi
 import androidx.compose.runtime.Composable
+import cafe.adriel.voyager.navigator.Navigator
+import cafe.adriel.voyager.transitions.FadeTransition
+import com.beepbeep.designSystem.ui.theme.BpTheme
+import org.thechance.common.di.initKoin
+import org.thechance.common.ui.login.LoginScreen
 
+
+@OptIn(ExperimentalAnimationApi::class)
 @Composable
 fun App() {
-    DesignApp()
+    initKoin()
```

Comment:
i'm facing a crash because of this line, i have two monitors in my setup when i'm trying to move the app from one monitor to the other app the is crashing,

and this is the error message: " A Koin Application has already been started "